### DISCUSSION TITLE
Discord

### DISCUSSION THREAD

**Discussion by ghost** [Pryaxis/TShock#2317] [Q&A] [answered]
Also why i can't join to the Thock ds server

**Comment by moisterrific** [comment, marked as answer]
It's working fine for me, are you sure you have the right link? It's the one in the readme.

![image](https://user-images.githubusercontent.com/57187883/118562019-f2d50e00-b739-11eb-8808-3df1ca599479.jpeg)

**Reply by ghost** [reply]
i don't know i can't join says error

**Reply by QuiCM** [reply]
You must have a phone linked to your discord account in order to join

**Reply by ghost** [reply]
i have phone number linked to my account it just says oops couldn't connet you to the server

**Reply by bartico6** [reply]
this message generally appears if you're banned on the server. post the image to confirm/deny this.

**Reply by ghost** [reply]
> this message generally appears if you're banned on the server. post the image to confirm/deny this.

![Screenshot_20210519-072243_Discord-Beta](https://user-images.githubusercontent.com/70816842/118756199-151e7700-b873-11eb-9957-3d25027947c4.jpg)

**Reply by hakusaro** [reply]
Which link did you click?

**Reply by ghost** [reply]
I used link in the readme

**Reply by drunderscore** [reply]
Then it would seem you are banned.

**Reply by ghost** [reply]
Well is there any ways to appeal?
